.circleci/config.yml

```diff
@@ -136,7 +136,9 @@ jobs:
           template: basic_fail_1
           branch_pattern: main
   run_specs:
+    parallelism: 4
     executor: test-executor
+    resource_class: small
```

Comment:
comment - thinking about it, we possible don't event need to put `resource_class: small` here as it's already defined in `executor: test-executor`